docs/developers/guides/coin_token_registration.md

```diff
@@ -13,19 +13,28 @@ Application-wise, the ERC-20 module allows DeFi protocols to seamlessly integrat
 - earning $OSMO staking rewards
 - taking part in governance proposals by voting with $ATOM
 
-Registering an interoperable ERC-20 means registering a new mapping between an ERC-20 token contract and a Cosmos Coin denomination, also known as a Token Pair. Token Pairs enable users to convert ERC-20 tokens into their native Cosmos Coin representation, and can only be created via a governance proposal.
+Registering Cosmos Coins and interoperable ERC-20 tokens means registering a new mapping between an ERC-20 token contract and a Cosmos Coin denomination, also known as a Token Pair. Token Pairs enable users to convert ERC-20 tokens into their native Cosmos Coin representation, and can only be created via a governance proposal.
 
-More information on ERC-20s can be found in [this blog post](https://medium.com/evmos/introducing-evmos-erc20-module-f40a61e05273), which introduced the ERC-20 Module on Evmos.
+More information on the `x/erc20` module can be found in [this blog post](https://medium.com/evmos/introducing-evmos-erc20-module-f40a61e05273), which introduced the ERC-20 Module on Evmos.
 
-To register an ERC-20, consider the following stages:
+To register a Cosmos Coin or an ERC-20, consider the following stages:
 
-1. [Drafting the ERC-20 Proposal](#drafting-the-erc-20-proposal)
-2. [Submitting the ERC-20 Proposal](#submitting-the-erc-20-proposal)
-3. [The On-Chain ERC-20 Proposal](#the-on-chain-erc-20-proposal)
+1. [Drafting the Registration Proposal](#drafting-the-registration-proposal)
+2. [Submitting the Registration Proposal](#submitting-the-registration-proposal)
+3. [The On-Chain Registration Proposal](#the-on-chain-registration-proposal)
 
-## Drafting the ERC-20 Proposal
+:::tip
```

Comment:
Remove tip formatting. These notes should be used for short explanations instead.